app/views/bookings/index.html.haml

```diff
@@ -0,0 +1,19 @@
+%table.table.table-condensed.table-striped.table-bordered.table-hover#bookings_table
+  %thead
+    %tr
+      %th
+        Applicant
+      %th
+        When
+      %th
+  %tbody
+    - @bookings.each do |booking|
+      - applicant = booking.applicant
+      %tr
+        %td
+          = link_to "#{applicant.display_name}", user_profile_path(applicant), data: { no_turbolink: true }
```

Comment:
I'm noticing `no_turbolink: true` is being used through the app. It'd be good to get to the bottom of understanding why or if this is needed and come up with a less duplicative solution. For example disabling turbolinks application-wide or fixing a page load JavaScript issue if that's the underlying problem.